src/crossgen2/src/Program.cs

```diff
@@ -147,6 +149,15 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 Console.ReadLine();
             }
 
+            if(_includeGenericsFromVersionBubble)
```

Comment:
Nit: `if` is not a function call. We put a space after it (and there's another occurence of it in SignatureBuilder.cs)

Reply:
Weird... I expected VS to take care of that :)